samples/openthread/cli/README.rst

```diff
@@ -518,6 +519,17 @@ To test the Thread 1.2 and Thread 1.3 features, complete the following steps:
             addresses: [fdde:ad00:beef:0:e0fc:dc28:1d12:8c2]
          Done
 
+Power consumption measurements
+==============================
+
+The Thread CLI sample can be used to perform power consumption measurements for Sleepy End Devices.
+After building and flashing with :file:`overlay-low_power.conf` and :file:`low_power.overlay` the device will start regular operation with UART console enabled.
+This allows for easy configuration of the device, specifically the SED polling period or the SSED CSL period, among other relevant parameters.
+Then, once the device becomes attached to a Thread Router it will automatically suspend UART operation and power down unused RAM.
```

Comment:
```suggestion
When the device becomes attached to a Thread Router it will automatically suspend UART operation and power down unused RAM.
```
Add an empty line above this line as well.